### PR TITLE
Created Arm Subsystem & RunArmsManually (Fixes #67)

### LINKED ISSUE #67: Basic arm control
Control the arm using the analog triggers on the gamepad
- [x] Create an Arms subsystem with a single VictorSP for motor control (use RobotMap to define which port the motor uses on the RoboRIO)
- [x] Add a method to the Arms class that sets the speed of the motor, taking a `double` parameter
- [x] Update the Robot class with the Arms system
- [x] Add a method to the OI to get the desired intake speed from the gamepad analog triggers. Right  trigger should lower the arms, left trigger should raise them (deadband should be applied to joystick value)
- [x] Add a "RunArmsManually" command created that sets the intake speed based on the desired value from the OI
- [x] Set the default command for the Arms subsystem to the RunArmsManually command

### PR DESCRIPTION
**Pull request by JCharante**

### CODE REVIEW

**Comment by rtfoley** [comment]
@JCharante a few suggestions:
- Have only one method in OI for getting the arm speed
- We don't need acceleration caps for the arms in that subsystem

**Comment by rtfoley** [comment]
Just a couple things:
- in the command, the requires(arm) should go in a constructor, not initialize()
- there are some extra blank lines inside methods (not between them, those are fine) you can get rid of

**Comment by wilstenholme** [comment]
A few more things:
- in OI, the axes should be variables from Axes enum in Gamepad
- applyDeadband() methods could be reduced into one method, possibly?
- either use Arm or Arms for consistency

**Comment by rtfoley** [comment]
@wilstenholme  I agree, "Arms" makes more sense to me. I think leaving the deadband code as-is works fine. It gives each trigger the full deadband.

**Comment by JCharante** [comment]
Since Gamepad.Axes.LEFT_TRIGGER.getNumber() returns an int and the ranges for the motors are (-1 to 1), wouldn't this not work, or if it did wouldn't it work horribly

File: src/main/java/org/usfirst/frc/team5687/robot/RobotMap.java [hunk 1]
```diff
@@ -39,4 +39,8 @@
     public static class Intake {
         public static int INTAKE_MOTOR = 1;
     }
+
+    public static class Arm {
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/RobotMap.java]: Change to 'Arms'

File: src/main/java/org/usfirst/frc/team5687/robot/OI.java [hunk 2]
```diff
@@ -87,6 +87,10 @@ public double getIntakeSpeed() {
         return Helpers.applyDeadband(joystick.getRawAxis(2), Constants.Deadbands.INTAKE_STICK);
     }
 
+    public double getArmsSpeed() {
+        return Helpers.applyDeadband(Gamepad.Axes.LEFT_TRIGGER.getNumber(), Constants.Deadbands.ARM) - Helpers.applyDeadband(Gamepad.Axes.RIGHT_TRIGGER.getNumber(), Constants.Deadbands.ARM);
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/OI.java]: You need to use `gamepad.getRawAxis(Gamepad.Axes.LEFT_TRIGGER)` inside the applyDeadband call in order to get the stick value. The current code just gets the axis layout number, not it's real-time control value.

File: src/main/java/org/usfirst/frc/team5687/robot/subsystems/Arms.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+public class Arms extends Subsystem {
+
+    private VictorSP armsMotor;
+
+    public Arms() {
+        armsMotor = new VictorSP(RobotMap.Arm.ARM_MOTOR);
+    }
+
+    public void setSpeed (double speed) {
+
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Arms.java]: Extra blank line isn't needed here.

File: src/main/java/org/usfirst/frc/team5687/robot/commands/RunArmsManually.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team5687.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team5687.robot.OI;
+import org.usfirst.frc.team5687.robot.Robot;
+import org.usfirst.frc.team5687.robot.subsystems.Arms;
+
+/**
+ * Created by John on 2/10/2016.
+ */
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/commands/RunArmsManually.java:11]: Add some test about the class purpose to this Javadoc string

File: src/main/java/org/usfirst/frc/team5687/robot/subsystems/Arms.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team5687.robot.subsystems;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc.team5687.robot.Constants;
+import org.usfirst.frc.team5687.robot.RobotMap;
+import org.usfirst.frc.team5687.robot.commands.RunArmsManually;
+
+/**
+ * Created by John on 2/10/2016.
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Arms.java:9]: Add some text about the class/ subsystem purpose here.

**Comment by rtfoley** [comment]
@JCharante regarding your question, the .getNumber() call just returns the layout number for that axis, not it's control value. You need to use `gamepad.getRawAxis(...)` for that.

**Comment by rtfoley** [comment]
Looks good to me, sending to @BenBernardCIS for 2nd review.

File: src/main/java/org/usfirst/frc/team5687/robot/Constants.java [hunk 6]
```diff
@@ -14,6 +14,7 @@
         public static final double DRIVE_STICK = 0.1;
         public static final double SHOOTER_WHEELS = 0.1;
         public static final double INTAKE_STICK = 0.1;
+        public static final double ARM = 0.1;
```
Comment (wilstenholme) [inline src/main/java/org/usfirst/frc/team5687/robot/Constants.java]: Change this to ARMS

**Comment by wilstenholme** [comment]
@rtfoley Looks good to me.

**Pull request merged**